com/edocti/jintro/lab01/NewString.java

```diff
@@ -0,0 +1,10 @@
+package com.edocti.jintro.lab01;
+public class NewString {
+	public static void main(String[] args) {
```

Comment:
Little thing: I think you use a tab size of 8 spaces or something. Not a huge problem, but it's usually better to stick to 4 or 2 (everywhere)